Brayton cycle demonstration (#20217)
idaholab/moose: modules/doc/content/newsletter/2022_02.md

- Axisymmetric and spherical coordinates.
- Generalized plane strain and plane stress formulations.

### Brayton cycle model demonstrated using the thermal hydraulics module

Examples demonstrating the startup and steady operation of a simple open-Brayton

> and closed-Brayton cycle was demonstrated using the

Reviewer: ```suggestion
and closed-Brayton cycle were created using the
```
